### DISCUSSION TITLE
Error: Command "npm run build" exited with SIGABRT

### DISCUSSION THREAD

**Discussion by fasanidev** [vercel/community#5048] [Help] [answered]
### Summary

Hi! From some days I've start to get this annoying error during a very simple next js project deployment which uses **prismic**
```
Creating an optimized production build ...
09:36:50.357 | corrupted size vs. prev_size
09:38:44.307 | Error: Command "npm run build" exited with SIGABRT
```

and honestly I don't even know where to look; anyone has a clue?

### Steps to Reproduce

-

**Comment by swarnava** [comment]
Hi,

Are you using latest Next.js and Node version? Can you reproduce without build cache?

**Comment by gilsmt** [comment]
Hey I'm getting the same issue but without any other message
Using turborepo, tried redeploying without cache multiple times.
The project was building correctly a few days ago, and it builds locally correctly
Using node 18.17
```
 ▲ Next.js 14.0.3
04:33:14.864 | Creating an optimized production build ...
04:33:15.470 | ✓ (pwa) Compiling for server...
04:33:15.499 | ✓ (pwa) Compiling for server...
04:33:15.501 | ✓ (pwa) Compiling for client (static)...
04:33:15.502 | ○ (pwa) Service worker: /vercel/path0/apps/web/public/sw.js
04:33:18.834 | 
04:33:21.958 | corrupted size vs. prev_size
04:35:59.684 | ERROR: command finished with error: command (/vercel/path0/apps/web) npm run build exited (-1)
04:35:59.684 | command (/vercel/path0/apps/web) npm run build exited (-1)
04:35:59.685 |  
04:35:59.685 | Tasks:    0 successful, 1 total
04:35:59.685 | Cached:    0 cached, 1 total
04:35:59.685 | Time:    2m47.003s
04:35:59.685 | Summary:    /vercel/path0/.turbo/runs/2Z10h01hn15WfQByfed0b1zFXp1.json
04:35:59.685 | Failed:    @web#build
04:35:59.685 |  
04:35:59.685 | ERROR  run failed: command  exited (1)
04:35:59.697 | Error: Command "NODE_OPTIONS="--max-old-space-size=8192" turbo run build" exited with 1
04:36:00.881
```

**Reply by gilsmt** [reply]
On another redeployment of the same build I got `free(): invalid next size (fast)` instead of `corrupted size vs. prev_size`

**Reply by gilsmt** [reply, marked as answer]
Looks like it was because I updated sharp https://github.com/lovell/sharp/issues/3870, https://github.com/vercel/nft/issues/371. Vercel seems to be having dependency issues
